content/so/0009-process-management.md

```diff
@@ -284,8 +284,8 @@ Isto permite ir "esquecendo" progressivamente os usos mais antigos do CPU.
 
 O Unix suporta ainda as seguintes chamadas de sistema:
 
-- `nice(int val)`: decrementa a prioridade val unidades. Apenas superutilizador pode invocar com val negativo (isto é, tornar o processo mais prioritário);
-- `getpriority(int which, int id)`: retorna prioridade de um processo ou grupo de processos;
+- `nice(int val)`: incrementa a prioridade val unidades. Apenas superutilizador pode invocar com val negativo (isto é, tornar o processo mais prioritário);
```

Comment:
De acordo com os slides é decrementado

Reply:
![image](https://user-images.githubusercontent.com/7467891/155529767-21aeee05-879c-40b8-b1b4-a1493ffc4960.png)
According to the man page (`man 3 nice`), the function **increases the value of nice**. However, the higher the value of `nice`, the lesser the priority of that process.
I think this should be clarified.